plots/dynamics/free_fall_acceleration.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+from sympy.physics.units import gravitational_constant
```

Comment:
sympy.physics.units is already imported as units. Use `units.gravitational_constant` to access this entity.